.changeset/rotten-ravens-nail.md

```diff
@@ -0,0 +1,5 @@
+---
+"@localyze-pluto/components": patch
```

Comment:
i dont think this is a patch, it's not really fixing anything... should we change it to `minor` ?